full_name.rb

```diff
@@ -0,0 +1,18 @@
+
+    full_name = []
+    print "What's your first name?"
```

Comment:
It's not visually pleasing when there's no space between the output and input, like in:

```
What's your first name?Daniel
```

One thing you could do is have the line break:

```suggestion
    puts "What's your first name?"
```

so that it's nicely spaced like so:

```
What's your first name?
Daniel
```

Reply:
Alternatively, use a colon and space like so:

```suggestion
    print "Please enter your first name: "
```

for

```
Please enter your first name: Daniel
```